docs/messenger/messenger.rst

```diff
@@ -0,0 +1,131 @@
+Messenger
+=========
+
+Messenger is a feature in Mautic 5.x that offers several significant benefits:
+
+.. _highlights:
+
+**Load distribution/control**
+    It enables you to balance and control the server load effectively.
+
+**Improved client response**
```

Comment:
What are we referring to here when we are talking about clients? Will marketers understand that?